.github/workflows/release.yml

```diff
@@ -174,6 +174,9 @@ jobs:
               run: |
                   pnpm install
                   pnpm run fmt
+            - name: "Types check"
+              run: |
+                  pnpm run typecheck
```

Comment:
Can you add this to

```
pnpm run lint
```
instead of being a new check?

Then we don't need to modify CI and devs already run `pnpm run lint`

Reply:
sure